.github/workflows/pull-request.yaml

```diff
@@ -211,10 +211,16 @@ jobs:
           location: ${{ secrets.GKE_ZONE }}
           credentials: ${{ secrets.GKE_SA_KEY }}
 
+      - name: Clone CLI code
+        uses: actions/checkout@v3
+        with:
+          path: 'tracetest-main'
+          ref: 'main'
       - name: Build CLI
         run: |
-          cd cli
+          cd tracetest-main/cli
           make build
+          cp tracetest ../../cli
```

Comment:
`tracetest-integration` doesn't support the new format yet. So I'm building the client from the main branch so we can send a compatible request to it